client/components/ProductDetails.jsx

```diff
@@ -9,11 +9,10 @@ class ProductDetails extends React.Component {
       products: []
     }
   }
-
   componentDidMount() {
     axios.get('/productDetails')
     .then((response) => {
```

Comment:
the airbnb guide requires that these thens get indented. i'm fine with the way you have it, but let's chat with the group so all agree on a standard.